### PR TITLE
feat: commute details section in my profile

### PR DESCRIPTION
**Pull request by Chethan-Fyle**
### Description
copilot:summary

### Walkthrough
copilot:walkthrough

## Clickup
https://app.clickup.com/t/8678ynw6v

## Code Coverage
Please add code coverage here

## UI Preview
Screenshot 2024-01-18 at 8 28 58 AM

### CODE REVIEW

File: src/app/fyle/my-profile/my-profile.page.ts [hunk 1]
```diff
@@ -67,12 +69,18 @@ export class MyProfilePage {
 
   infoCardsData: CopyCardDetails[];
 
+  commuteDetails: any;
```
Comment (Dimple16) [inline src/app/fyle/my-profile/my-profile.page.ts:72]: why `any`?

File: src/app/fyle/my-profile/my-profile.page.ts [hunk 2]
```diff
@@ -172,6 +181,20 @@ export class MyProfilePage {
     const orgUserSettings$ = this.orgUserSettingsService.get().pipe(shareReplay(1));
     this.org$ = this.orgService.getCurrentOrg();
     const orgSettings$ = this.orgSettingsService.get();
+    this.eou$.subscribe((eou) => {
+      const queryParams = {
+        params: {
+          user_id: `eq.${eou.us.id}`,
+        },
+      };
+      return this.spenderService.get('/employees', queryParams).subscribe((res: any) => {
```
Comment (Dimple16) [inline src/app/fyle/my-profile/my-profile.page.ts:190]: we shouldn't be using `any`

File: src/assets/svg/arrow-tail-up-down.svg [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Dimple16) [inline src/assets/svg/arrow-tail-up-down.svg:1]: can we please add these icons to the `icon.module.ts` file too?

File: src/app/fyle/my-profile/my-profile.page.ts [hunk 4]
```diff
@@ -172,6 +181,20 @@ export class MyProfilePage {
     const orgUserSettings$ = this.orgUserSettingsService.get().pipe(shareReplay(1));
     this.org$ = this.orgService.getCurrentOrg();
     const orgSettings$ = this.orgSettingsService.get();
+    this.eou$.subscribe((eou) => {
```
Comment (Dimple16) [inline src/app/fyle/my-profile/my-profile.page.ts:184]: are we adding the unit tests for this change separately?

**Review by Dimple16 — CHANGES_REQUESTED** [review]
added comments

File: src/app/fyle/my-profile/my-profile.page.ts [hunk 5]
```diff
@@ -189,11 +204,38 @@ export class MyProfilePage {
         this.orgUserSettings = res.orgUserSettings;
         this.orgSettings = res.orgSettings;
 
+        this.isMileageEnabled = this.orgSettings.mileage?.allowed && this.orgSettings.mileage.enabled;
+        this.isCommuteDeductionEnabled =
+          this.orgSettings.commute_deduction_settings?.allowed && this.orgSettings.commute_deduction_settings?.enabled;
```
Comment (Dimple16) [inline src/app/fyle/my-profile/my-profile.page.ts:209]: `this.orgSettings.commute_deduction_settings?.allowed && this.orgSettings.commute_deduction_settings.enabled;`

**Review by Dimple16 — APPROVED** [review]

**Review by OmkarJ13 — APPROVED** [review]

**Pull request merged**